liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/CreateServerMojo.java

```diff
@@ -37,6 +41,40 @@
      */
     private String template;
     
+    /**
+     * Packages to install. One of "all", "dependencies" or "project".
+     * 
+     * @parameter property="installAppPackages" default-value="dependencies"
+     * @readonly
+     */
+    private String installAppPackages = "dependencies";
+    
```

Comment:
So far I don't have a solution, unless we promote the installApps pararmeters to common parameters which are defined in BasicSupport.java.  If you like this approach, I can open a separate  issue for refactoring the installApps parameters.